posthog/src/main/java/com/posthog/java/PostHog.java

```diff
@@ -194,4 +199,66 @@ private JSONObject getEventJson(String event, String distinctId, Map<String, Obj
         }
         return eventJson;
     }
+
+    /**
+     * 
+     * @param featureFlag which uniquely identifies your feature flag
+     *
+     * @param distinctId which uniquely identifies your user in your database. Must
+     *                   not be null or empty.
+     */
+    public boolean isFeatureFlag(String featureFlag, String distinctId) {
```

Comment:
Ah, mind adding a @return to the doc string as well please? Noticed some warnings in the CI for this.


```suggestion
    public boolean isFeatureFlagEnabled(String featureFlag, String distinctId) {
```